domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -91,6 +99,17 @@ class TopicListController @Inject constructor(
     )
   }
 
+  /**
+   * Returns the list of [TopicSummary]s currently tracked by the app, possibly up to
+   * [EVICTION_TIME_MILLIS] old.
+   */
+  fun getComingSoonTopicList(): DataProvider<ComingSoonTopicList> {
```

Comment:
Why do we need both a top-level getComingSoonTopicList and getRecommendedActivityList?

Reply:
This is not required. It was old implemented before. Now implementation is changed. I will remove it.